app/actions/webhook.go

```diff
@@ -0,0 +1,137 @@
+package actions
+
+import (
+	"context"
+	"github.com/getfider/fider/app/models/cmd"
+	"github.com/getfider/fider/app/models/entity"
+	"github.com/getfider/fider/app/models/enum"
+	"github.com/getfider/fider/app/pkg/bus"
+	"github.com/getfider/fider/app/pkg/validate"
+)
+
+type CreateEditWebhook struct {
+	Name        string             `json:"name"`
+	Type        enum.WebhookType   `json:"type"`
+	Status      enum.WebhookStatus `json:"status"`
+	Url         string             `json:"url"`
+	Content     string             `json:"content"`
+	HttpMethod  string             `json:"http_method"`
+	HttpHeaders entity.HttpHeaders `json:"http_headers"`
+}
+
+// IsAuthorized returns true if current user is authorized to perform this action
+func (action *CreateEditWebhook) IsAuthorized(_ context.Context, user *entity.User) bool {
+	return user != nil && user.IsAdministrator()
+}
+
+// Validate if current model is valid
+func (action *CreateEditWebhook) Validate(ctx context.Context, _ *entity.User) *validate.Result {
+	result := validate.Success()
+
+	if action.Name == "" {
+		result.AddFieldFailure("name", "Name is required.")
+	} else if len(action.Name) > 60 {
+		result.AddFieldFailure("name", "Name must have less than 100 characters.")
```

Comment:
60 :)

Reply:
Damn I must be super tired to miss that 😆